resources/views/frontend/trainee/trainee-profile.blade.php

```diff
@@ -5,51 +5,86 @@
 @extends($layout)
 
 @section('title')
-{{$siteSettingInfo->site_title}} :: {{__('generic.youth_profile')}}
+    {{ $siteSettingInfo->site_title }} :: {{ __('generic.youth_profile') }}
 @endsection
 
 @push('css')
     <style>
         .profile-info-p {
             line-height: normal;
         }
-    </style>
 
+        .custom-button-style {
+            background-color: #671688;
+            color: white !important;
+            border-radius: 12px;
+        }
+
+        .custom-button-style:hover {
+            background-color: #ffffff;
+            color: black !important;
+            border-color: #671688;
+        }
+
+        @media screen and (max-width: 767px) {
+            .trainee-info {
+                text-align: center;
+                margin-top: 1rem;
+            }
+        }
+
+        .custom-edit-button {
+            border-color: #671688 !important;
+            color: black !important;
+        }
+
+        .custom-edit-button:hover {
+            border-color: #671688 !important;
+            background-color: #671688 !important;
+            color: white !important;
+        }
+    </style>
 @endpush
 
 @section('content')
     <div class="container-fluid">
         <div class="row trainee-profile justify-content-center" id="trainee-profile">
 
-            <div class="col-md-10 mt-2 personal-info-section">
+            <div class="col-md-10 mt-3 pt-1 personal-info-section">
                 <div class="card">
                     <div class="card-header">
                         <div class="card-title">
-                            <h4 class="font-weight-bolder">Personal Information</h4>
+                            <h4 style="transform: translateY(30%)">Personal Information</h4>
                         </div>
 
                         <div class="card-tools">
```

Comment:
```
<a href="{{ route('frontend.edit-personal-info') }}"
                                class="card-tools btn btn-sm btn-primary custom-button-style btn-rounded px-4 pt-2 pb-1 font-weight-bold">
                                <!-- <i class="fas fa-plus-circle"></i> -->
                                {{ __('admin.common.edit') }}
                            </a>
                       </div>
                    <div class="card-body">
                        <div class="row justify-content-flex-start">
                            <div class="col-md-4 col-lg-3 col-xl-2 mr-xl-5 mr-lg-4 ml-lg-4 d-flex justify-content-center">
```
`Remove <div class="card-tools">` and add the above piece of code.
Next after ` <div class="card-body">`
```
<div class="row justify-content-flex-start">
                            <div class="col-md-4 col-lg-3 col-xl-2 mr-xl-5 mr-lg-4 ml-lg-4 d-flex justify-content-center">
```